lib/nrf_modem_lib/nrf_modem_lib_trace.c

```diff
@@ -127,9 +136,12 @@ int nrf_modem_lib_trace_stop(void)
 	__ASSERT(!k_is_in_isr(),
```

Comment:
This assert can be removed now.

Reply:
We have talked about adding the AT-command back when we are able to detect that the modem has crashed, so I think it's best to keep this.

Reply:
Cant we add this back then?